src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/annotations/AExpect.java

```diff
@@ -53,7 +53,8 @@
    * Currently supported places to look for RCA outputs are the SQLite file or by hitting the rest endpoint.
    */
   enum Type {
-    REST_API
+    REST_API,
+    DB_QUERY
```

Comment:
Why are we making it a generic `DB_QUERY`? Can we instead make it an `Action` since that is what we persist in the table. And use `from_summary()` methods to create actions from summary. 
The summary will contain entire details of the action which may create some redundancy with existing columns in the persisted action table, but that should be fine.
I think this IT PR should also include the change to create actions from summary strings, and use them for validation.

Reply:
Retaining DB_QUERY to enable querying the sqlite tables. With regard to Action, after Action API is implemented (in progress presently), we will move to use the REST_API instead (like rca's ) instead of querying the tables.